src/components/ConfigurationRow.tsx

```diff
@@ -144,24 +145,23 @@ export const getConfigurationRow = ({
       return (
         <>
           {overrideValue}
-          {!disabled && (
-            <Button
-              onClick={() => {
-                ensureEditMode(formik);
-                if (!isOverridden) {
-                  enableOverride();
-                }
-                focusOverride();
-              }}
-              className="u-no-margin--bottom"
-              type="button"
-              appearance="base"
-              title={isOverridden ? "Edit" : "Create override"}
-              hasIcon
-            >
-              <Icon name="edit" />
-            </Button>
-          )}
+          <Button
+            onClick={() => {
+              ensureEditMode(formik);
+              if (!isOverridden) {
+                enableOverride();
+              }
+              focusOverride();
+            }}
+            className="u-no-margin--bottom"
+            type="button"
+            appearance="base"
+            title={isOverridden ? "Edit" : "Create override"}
+            disabled={disabled}
```

Comment:
Since the disabled button is now shown, is it worth having a title or explanation on hover detailing why the button is disabled? Same for other instance of this.

Reply:
Ah yes, good idea. Will add the reason as a title.